From 471e22d6bab682a11b39eb0a68197e620f271e4c Mon Sep 17 00:00:00 2001
From: Brendan Furneaux <brendan.furneaux@gmail.com>
Date: Sat, 12 Oct 2024 11:18:49 +0200
Subject: [PATCH] add -lpthread on windows

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index 4566d1b..07ed73d 100755
--- a/configure
+++ b/configure
@@ -97,7 +97,7 @@ if [ -n "$WINDOWS" ]; then
     OBJECTS="${OBJECTS} arch/windows/wmi.o"
 
     LIBRARIES="psapi kernel32 advapi32 shell32 netapi32 iphlpapi wtsapi32"
-    LIBRARIES="${LIBRARIES} ws2_32 PowrProf Pdh"
+    LIBRARIES="${LIBRARIES} ws2_32 PowrProf Pdh pthreads"
 
     TARGETS=interrupt
 
-- 
2.34.1

